### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by MatteoRobbiati** [qiboteam/boostvqe#1]
VQE tasks:

- [x] define optimizer and, more in general, optimization strategy. Default now is Powell optimizer.
- [x] Define a circuit ansatz for the VQE. Now we are setting the same ansatz of [this work](https://arxiv.org/pdf/2002.06210.pdf), which is also the same we have in [this `qibo` example](https://github.com/qiboteam/qibo/tree/master/examples/aavqe). 
- [x] define (if we want it) a way to perform hyper-optimization of the circuit's architecture;
- [x] plotting functions in a `plotscripts` folder;
- [x] write a script to quantify the gradients magnitude, in order to evaluate the trainability of the models with screenshots of the loss function landscape.
- [x] have a clear methodology for VQE training and a clear mapping out when a BP is reached

DBI boosting taks:

- [x] have transpiler ready
- [x] PR transpiler
- [x] do gate count
- [x] explore how far one can get
